### ISSUE TITLE
 err_te is 1

### ISSUE THREAD

**Issue by severusbunny** [mravanelli/pytorch-kaldi#256]
I use my dataset and in training i am getting test error 100%,how can I fix this problem?

res.res is shown as following :
epoch 0, loss_tr=8.295877 err_tr=0.996953 loss_te=12.132339 err_te=1.000000 err_te_snt=1.000000
epoch 8, loss_tr=6.212218 err_tr=0.932266 loss_te=15.733055 err_te=1.000000 err_te_snt=1.000000
epoch 16, loss_tr=5.513986 err_tr=0.864531 loss_te=18.980804 err_te=1.000000 err_te_snt=1.000000

And I use test set as train set
res.res is shown as following:
epoch 0, loss_tr=6.637052 err_tr=0.990234 loss_te=6.063669 err_te=0.984697 err_te_snt=0.973500
epoch 8, loss_tr=3.301901 err_tr=0.710049 loss_te=3.365894 err_te=0.715733 err_te_snt=0.241827
epoch 16, loss_tr=2.495244 err_tr=0.562266 loss_te=2.602354 err_te=0.585831 err_te_snt=0.141579
epoch 24, loss_tr=2.032088 err_tr=0.470723 loss_te=2.049465 err_te=0.477058 err_te_snt=0.076674
epoch 32, loss_tr=1.768699 err_tr=0.413418 loss_te=1.816929 err_te=0.428155 err_te_snt=0.064197
epoch 40, loss_tr=1.570135 err_tr=0.373398 loss_te=1.538213 err_te=0.369554 err_te_snt=0.046786
epoch 48, loss_tr=1.439105 err_tr=0.344004 loss_te=1.490549 err_te=0.357521 err_te_snt=0.038102
